Action.swift

```diff
@@ -75,6 +88,15 @@ public extension Action {
 public extension Action {
 
     public func execute(input: Input) -> Observable<Element> {
+        // Send input to `self.inputs` to see all inputs in `self.inputs`.
```

Comment:
I'm wondering if we can simplifying things here a little and get rid of `skipsInputs`. I don't see a way, but we should document how this works and why.